### ISSUE TITLE
provide keyboard accessibility

### ISSUE THREAD

**Issue by Asmatzaile** [satelllte/react-knob-headless#84]
If the user interacts with the knob with the keyboard, it is not possible right now to change its values. The knob can be focused via the prop includeIntoTabOrder={true}, but it cannot be changed with the keyboard keys. Providing the accessibility is quite straightforward, though, by changing the following to true: https://github.com/satelllte/react-knob-headless/blob/b4ac2bf206ce7d9b2f2924ddb26d6a56b542e883/packages/react-knob-headless/src/KnobHeadless.tsx#L142
What's the reason why it was set to false in the first place?
Thanks :)

**Comment by satelllte** [comment]
Hi. It is set to `false` because there's a separate hook for that: `useKnobKeyboardControls`

https://github.com/satelllte/react-knob-headless/blob/main/packages/react-knob-headless/src/useKnobKeyboardControls.ts

https://react-knob-headless.pages.dev/

**Comment by Asmatzaile** [comment]
Thank you! Completely missed that
